recipes/geant4/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "10.5.0" %}
+
+package:
+    name: geant4
+    version: {{ version }}
+source:
+    url: http://cern.ch/geant4-data/releases/geant4.10.05.tar.gz
```

Comment:
Using something like this will make it easy to update without risking that the version number and sources get out of sync:

```yaml
{% set name = "geant4" %}
{% set version = "10.05" %}
{% set patch_version = "00" %}

package:
  name: {{ name|lower }}
  version: {{ version }}.{{ patch_version }}

source:
{%- if patch_version == "00" %}
  url: http://geant4-data.web.cern.ch/geant4-data/releases/{{ name }}.{{ version }}.tar.gz
{%- else %}
  url: http://geant4-data.web.cern.ch/geant4-data/releases/{{ name }}.{{ version }}.p{{ patch_version }}.tar.gz
{%- endif %}
  sha256: 2a86499d8327abc68456e5d7fc0303824e5704322291b331857cf4042286656e
```

Reply:
Not a fan of this as the version will then contain the leading zero in the minor version number, and I need the version number in its current form in `run_test.sh`.

Reply:
When fixing the testing issue I also edited this to pad the zeros only for the URLs. What do you think of this version?

Reply:
That's some nice jinja templating! Neat!